src/components/intern-app.html

```diff
@@ -31,18 +31,45 @@
       }
     </style>
     <header>
-      <user-list></user-list>
-      <create-user></create-user>
-      <edit-user></edit-user>
+      <create-user storage="[[storage]]"></create-user>
+      <user-list users="[[users]]" storage="[[storage]]"></user-list>
     </header>
   </template>
 
   <script type="module">
     import { Element as PolymerElement } from '@banno/polymer/polymer-element.js';
+    import { LocalStorage } from '../api/localStorage.js';
+
+    const storage = new LocalStorage();
+    
     class InternApp extends PolymerElement {
       static get is() { return 'intern-app'; }
-    }
+      
+      static get properties() {
+        return {
+          storage: {
+            value: () => storage
+          },
+          users: {
+            value: () => []
+          }
+        }
+      }
+
+      connectedCallback() {
+        super.connectedCallback();
+        this.users = this.storage.getAllUsers();
+      }
 
+      ready() {
+        super.ready();
+        this.addEventListener('updateUserList', this.updateUserList);
+      }
+      
+      updateUserList(e) {
+        this.users = e.detail.data.updatedUsers;
```

Comment:
rather than running `this.storage.getAllUsers()` before/while dispatching `updateUserList`, and then setting `this.users` to `e.detail.data.updatedUsers`, just watch for the call _here_ and then make the `this.storage.getAllUsers()` here as well. So everything in `intern-app` will stay the same, other than this line.

```suggestion
        this.users = this.storage.getAllUsers();
```

then your message just needs to dispatch, let the component catching it worry about updating the users list

```
        this.dispatchEvent(new CustomEvent('updateUserList', { bubbles: true, composed: true }));
```

Reply:
Done